### ISSUE TITLE
Interpolation in Analog Input Configs 

### ISSUE THREAD

**Issue by JaimeLeon2** [MobiFlight/MobiFlight-Connector#1203]
**Is your feature request related to a problem? Please describe.**
Currently interpolation is only available in output configs. It would be very convenient to have this feature in Analog Input Configs to be able to calibrate potentiometers. 

**Describe the solution you'd like**
Create a new interpolation function for analog input, similar to the one in output configs. Sending the interpolated value directly to the Sim. This will allow direct calibration of potentiometer values to send to the Sim. 

**Describe alternatives you've considered**
Presently, to use interpolation in Analog Input requires sending the potentiometer value to an output config, using the interpolation function, to then reroute the value back to an input. 

**Additional context**
Current method is awkward, difficult for beginners and can only be considered as a workaround.  A new analog input interpolation would streamline pot calibrations.

**Comment by DocMoebiuz** [comment]
I agree, I have thought about it. I am wondering how we could bring a similar concept like we have it for outputs: `read > modify > display` to the input config side.

this would have to happen inside an an input action as a general mean before the input action gets executed. Basically as a modify of the `@` value.

**Comment by JaimeLeon2** [comment]
Maybe a new calibration tab in the Analog Input config wizard?
